sdk/batch/azure-batch/tests/test_batch.py

```diff
@@ -69,26 +70,26 @@ def get_redacted_key(key):
     return redacted_value
 
 
-class RecordingRedactor(RecordingProcessor):
-    """Removes keys from test recordings"""
+# class RecordingRedactor(RecordingProcessor):
```

Comment:
I assume you'll clean up all this commented out code

Reply:
Cleaned up comments